src/index.js

```diff
@@ -0,0 +1,26 @@
+const repl = require("repl");
+
+function getRandomNumber (max) {
```

Comment:
optionally you can add a check to ensure that a number has actually been provided